formhero/forms/permission_classes.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.permissions import BasePermission
+from formhero.apps.models import App
+
+
+class HostPermission(BasePermission):
+    def has_permission(self, request, view):
+        hosts = App.objects.values_list('host_list').get(pk=request.data['app'])[0]
+        thishost = request.get_host()
```

Comment:
@szymonzmyslony Use underscored variable names.
`this_host` is easier to read, and easier to spot if it has been misspelled.